src/sdk/sql_cluster_router.cc

```diff
@@ -625,6 +625,31 @@ bool SQLClusterRouter::ShowDB(std::vector<std::string>* dbs, hybridse::sdk::Stat
     }
     return true;
 }
+ 
+std::vector<std::string> SQLClusterRouter::GetAllTables(hybridse::sdk::Status* status){
+
+    std::vector<std::string> dbs;
+    std::vector<std::string> all_table;
+    bool ok = SQLClusterRouter::ShowDB(&dbs, status);
+    if (!ok) {
+        return all_table;
```

Comment:
print status to warning log

Reply:
in line 633 I've called the class member function `ShowDB`, the warning log of status (line620-624) may be already printed?

Reply:
ShowDB rpc can be remove too. DBSDK has the db-table-tableinfo map

Reply:
updated.